### DISCUSSION TITLE
More flexible file name conventions

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#4056] [Enhancement Queue]
Just an idea I thought I’d log here:

Allow Eleventy to search `*.engineName.someOtherFileExtension`

e.g. `test.liquid.css` would output to `test.css` and transform using liquid but still keep syntax highlighting for a CSS file.

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by Ryuno-Ki** [comment]
This always reminds me on [preprocessing of the assets pipeline in Ruby on Rails](https://guides.rubyonrails.org/asset_pipeline.html#preprocessing)!
